docs/dev_guide/diagnostic_ref/dpct1029.rst

```diff
@@ -25,3 +25,35 @@ Suggestions to Fix
 ------------------
 
 Review the logic and adjust it manually.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo(size_t *value, cudaLimit limit){
+      cudaDeviceGetLimit(value, limit);
+    }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo(size_t *value, cudaLimit limit){

```

Comment:
as a fix example,  let's be more specific on fix which limit, like cudaLimitPrintfFifoSize.

Reply:
done